src/components/UserActions/Strings.ts

```diff
@@ -17,8 +17,8 @@ Translation.register(Language.English, {
     UserActivity_other_event: 'Other Event',
     UserActivity_other_events: 'Other Events',
 
-    UserActivity_search: 'User Query',
-    UserActivity_query: 'Query',
+    UserActivity_search: 'Query',
+    UserActivity_query: 'User Query',
```

Comment:
What's changed exactly ?

Reply:
The label tagged in the UserActivity component.